src/h5web/providers/jupyter/api.ts

```diff
@@ -138,4 +127,33 @@ export class JupyterStableApi implements ProviderAPI {
       kind: EntityKind.Unresolved,
     };
   }
+
+  protected async processBaseEntity(
+    path: string,
+    response: JupyterMetaResponse
+  ): Promise<JupyterBaseEntity> {
+    // TODO: To fix once we have `attributeCount`
+    const attrReponse = await this.fetchAttributes(path);
+    const attributes = Object.entries(attrReponse).map(([name, value]) =>
+      // TODO: fix this once I can infer the type of attributes
+      makeStrAttr(name, value)
+    );
+
+    return {
+      name: response.name,
+      path,
+      attributes,
+    };
+  }
+
+  protected async processContents(
+    path: string,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    response: JupyterMetaGroupResponse
+  ): Promise<JupyterContentGroupResponse> {
+    const contents = await this.fetchContents(path);
+    assertGroupContent(contents);
+
+    return contents;
+  }
```

Comment:
I encapsulated the varying process parts into these functions that are overriden in the dev API.

Once these are no longer needed in the dev API, we can inline them once again.